### PR TITLE
Put the profile on the users page.

### PR DESCRIPTION
**Pull request by kevincox**

### CODE REVIEW

**Comment by matthewcrooke1** [comment]
That makes a lot more sense :volcano:

**Comment by GoodOlChuck** [comment]
fums up :+1:. Is following working now?

**Comment by kevincox** [comment]
I didn't click the button but I can see it.

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -22,12 +22,13 @@ def logout
 
   def home
   end
-  def profile 
-    if current_user.nil?
-       redirect_to :login 
+  
+  def profile
+    if current_user
+      redirect_to current_user
```
Comment (matthewcrooke1) [inline app/controllers/sessions_controller.rb:28]: Just a question, what does redirect_to current_user do differently than just giving back current_user

Comment (kevincox) [inline app/controllers/sessions_controller.rb:28]: Does returning `current_user` redirect there? I'm pretty sure the result of this method is ignored.

**Pull request merged**